phabricatoremails/templates/html/normal/accepted-as-author.html.jinja2

```diff
@@ -0,0 +1,39 @@
+{% import '_macros.html.jinja2' as macros %}
+{{ macros.head() }}
+<div class="event-content">
+
+    {% call macros.preview(event.is_ready_to_land, unique_number) %}
+        &#x2714; {{ actor_name }} accepted this revision {{- event | commentsummary }}.
```

Comment:
As mentioned elsewhere, these unicode characters should be defined in a dictionary with human readable names, and passed to the template in the context.